### PR TITLE
refactor: change engine name to color for output message

### PR DESCRIPTION
**Pull request by gahtan-syarif**
`Finished game 9 (random_move_1 vs random_move_2): 1-0 {Black got checkmated}`

fix https://github.com/Disservin/fast-chess/issues/532

### CODE REVIEW

**Comment by vondele** [comment]
Note 'White' vs 'white'

File: app/src/matchmaking/match/match.cpp [hunk 1]
```diff
@@ -143,6 +143,12 @@ bool Match::playMove(Player& us, Player& them) {
     const auto gameover = board_.isGameOver();
     const auto name     = us.engine.getConfig().name;
 
+
+    std::string color;
+
+    if (board_.sideToMove() == chess::Color::WHITE) color = "White";
+    else color = "Black";
```
Comment (Disservin) [inline app/src/matchmaking/match/match.cpp:150]: `auto color = ~board_.sideToMove() == chess::Color::WHITE ? "White" : "Black";`

Comment (gahtan-syarif) [reply app/src/matchmaking/match/match.cpp]: fixed 👍

File: app/src/matchmaking/match/match.hpp [hunk 2]
```diff
@@ -161,7 +161,7 @@ class Match {
 
     inline static constexpr char INSUFFICIENT_MSG[]     = "Draw by insufficient material";
     inline static constexpr char REPETITION_MSG[]       = "Draw by 3-fold repetition";
-    inline static constexpr char ILLEGAL_MSG[]          = " made an illegal move";
+    inline static constexpr char ILLEGAL_MSG[]          = " makes an illegal move";
```
Comment (gahtan-syarif) [inline app/src/matchmaking/match/match.hpp:164]: changed to be consistent with cutechess https://github.com/cutechess/cutechess/blob/1071d84cf272bd7deca0964336bf02e367e2b22b/projects/lib/src/board/result.cpp#L121

**Comment by vondele** [comment]
This isn't yet giving the right results. See how it writes:
`0-1 {White wins by adjudication}`  which would be a `Black wins ..` or a `1-0`.

fast-chess:
```
Starting tournament...
Started game 1 of 932 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0)
Started game 2 of 932 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0)
Started game 3 of 932 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0)
Started game 4 of 932 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0)
Finished game 3 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0): 0-1 {White wins by adjudication}
Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 1 - 0 - 0 [1.000] 1
Started game 5 of 932 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0)
Finished game 1 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1-0 {Black wins by adjudication}
Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 2 - 0 - 0 [1.000] 2
Started game 6 of 932 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0)
Finished game 4 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0): 0-1 {White wins by adjudication}
Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 2 - 1 - 0 [0.667] 3
Started game 7 of 932 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0)
Finished game 6 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1/2-1/2 {Draw by adjudication}
Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 2 - 1 - 1 [0.625] 4
Started game 8 of 932 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0)
Finished game 7 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1-0 {Black wins by adjudication}
Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 2 - 2 - 1 [0.500] 5
Started game 9 of 932 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0)
Finished game 2 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1-0 {Black wins by adjudication}
Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 2 - 3 - 1 [0.417] 6
Started game 10 of 932 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0)
Finished game 5 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1-0 {Black wins by adjudication}
Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 3 - 3 - 1 [0.500] 7
Started game 11 of 932 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0)
Finished game 8 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1-0 {Black wins by adjudication}
Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 4 - 3 - 1 [0.562] 8
Started game 12 of 932 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0)
```
cutechess-cli
```
Started game 6 of 466 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0)
Finished game 1 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1-0 {White wins by adjudication}
Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 2 - 0 - 1  [0.833] 3
Started game 7 of 466 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0)
Finished game 2 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1/2-1/2 {Draw by insufficient mating material}
Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 2 - 0 - 2  [0.750] 4
Started game 8 of 466 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0)
Finished game 7 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1-0 {White wins by adjudication}
Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 3 - 0 - 2  [0.800] 5
Started game 9 of 466 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0)
Finished game 5 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0): 0-1 {Black wins by adjudication}

```

**Comment by gahtan-syarif** [comment]
> This isn't yet giving the right results. See how it writes: `0-1 {White wins by adjudication}` which would be a `Black wins ..` or a `1-0`.
> 
> fast-chess:
> 
> ```
> Starting tournament...
> Started game 1 of 932 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0)
> Started game 2 of 932 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0)
> Started game 3 of 932 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0)
> Started game 4 of 932 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0)
> Finished game 3 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0): 0-1 {White wins by adjudication}
> Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 1 - 0 - 0 [1.000] 1
> Started game 5 of 932 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0)
> Finished game 1 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1-0 {Black wins by adjudication}
> Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 2 - 0 - 0 [1.000] 2
> Started game 6 of 932 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0)
> Finished game 4 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0): 0-1 {White wins by adjudication}
> Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 2 - 1 - 0 [0.667] 3
> Started game 7 of 932 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0)
> Finished game 6 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1/2-1/2 {Draw by adjudication}
> Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 2 - 1 - 1 [0.625] 4
> Started game 8 of 932 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0)
> Finished game 7 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1-0 {Black wins by adjudication}
> Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 2 - 2 - 1 [0.500] 5
> Started game 9 of 932 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0)
> Finished game 2 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1-0 {Black wins by adjudication}
> Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 2 - 3 - 1 [0.417] 6
> Started game 10 of 932 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0)
> Finished game 5 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1-0 {Black wins by adjudication}
> Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 3 - 3 - 1 [0.500] 7
> Started game 11 of 932 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0)
> Finished game 8 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1-0 {Black wins by adjudication}
> Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 4 - 3 - 1 [0.562] 8
> Started game 12 of 932 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0)
> ```
> 
> cutechess-cli
> 
> ```
> Started game 6 of 466 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0)
> Finished game 1 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1-0 {White wins by adjudication}
> Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 2 - 0 - 1  [0.833] 3
> Started game 7 of 466 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0)
> Finished game 2 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1/2-1/2 {Draw by insufficient mating material}
> Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 2 - 0 - 2  [0.750] 4
> Started game 8 of 466 (Base-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs New-ee6fc7e38b4aeef44862159215a56d97122f59a0)
> Finished game 7 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0): 1-0 {White wins by adjudication}
> Score of New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0: 3 - 0 - 2  [0.800] 5
> Started game 9 of 466 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0)
> Finished game 5 (New-ee6fc7e38b4aeef44862159215a56d97122f59a0 vs Base-ee6fc7e38b4aeef44862159215a56d97122f59a0): 0-1 {Black wins by adjudication}
> ```

ahh yeah my bad

**Comment by gahtan-syarif** [comment]
fixed 👍

**Pull request merged**